tests/test_mathematical.py

```diff
@@ -634,25 +635,75 @@ def test_invalid_shape(self, shape):
             dpnp.trunc(dp_array, out=dp_out)
 
 
-class TestPower:
+class TestAdd:
+    @pytest.mark.parametrize("dtype", get_all_dtypes(no_none=True))
+    def test_power(self, dtype):
```

Comment:
```suggestion
    def test_add(self, dtype):
```
Can you add tests for `__iadd__` function?

Reply:
> Can you add tests for `__iadd__` function?

It's already covered in `tests/test_usm_type.py`